savedstate/savedstate/api/desktop/savedstate.api

```diff
@@ -1,28 +1,102 @@
+	public static final fun getLongArray-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;)[J
+	public static final fun getLongArrayOrElse-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;Lkotlin/jvm/functions/Function0;)[J
+	public static final fun getLongOrElse-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;Lkotlin/jvm/functions/Function0;)J
+	public static final fun getSavedState-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;)Landroidx/savedstate/SavedState;
+	public static final fun getSavedStateOrElse-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;Lkotlin/jvm/functions/Function0;)Landroidx/savedstate/SavedState;
+	public static final fun getString-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;)Ljava/lang/String;
+	public static final fun getStringArray-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;)[Ljava/lang/String;
+	public static final fun getStringArrayOrElse-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;Lkotlin/jvm/functions/Function0;)[Ljava/lang/String;
+	public static final fun getStringList-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;)Ljava/util/List;
+	public static final fun getStringListOrElse-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;Lkotlin/jvm/functions/Function0;)Ljava/util/List;
+	public static final fun getStringOrElse-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;Lkotlin/jvm/functions/Function0;)Ljava/lang/String;
+	public fun hashCode ()I
+	public static fun hashCode-impl (Landroidx/savedstate/SavedState;)I
+	public static final fun isEmpty-impl (Landroidx/savedstate/SavedState;)Z
+	public static final fun isNull-impl (Landroidx/savedstate/SavedState;Ljava/lang/String;)Z
+	public static final fun size-impl (Landroidx/savedstate/SavedState;)I
+	public static final fun toMap-impl (Landroidx/savedstate/SavedState;)Ljava/util/Map;
+	public fun toString ()Ljava/lang/String;
+	public static fun toString-impl (Landroidx/savedstate/SavedState;)Ljava/lang/String;
+	public final synthetic fun unbox-impl ()Landroidx/savedstate/SavedState;
+}
+
+public final class androidx/savedstate/SavedStateReaderKt {
+	public static final fun keyNotFoundError (Ljava/lang/String;)Ljava/lang/Void;
+	public static final fun valueNotFoundError (Ljava/lang/String;)Ljava/lang/Void;
+}
+
 public final class androidx/savedstate/SavedStateRegistry {
-	public final fun consumeRestoredStateForKey (Ljava/lang/String;)Landroidx/core/bundle/Bundle;
+	public final fun consumeRestoredStateForKey (Ljava/lang/String;)Landroidx/savedstate/SavedState;
 	public final fun getSavedStateProvider (Ljava/lang/String;)Landroidx/savedstate/SavedStateRegistry$SavedStateProvider;
 	public final fun isRestored ()Z
 	public final fun registerSavedStateProvider (Ljava/lang/String;Landroidx/savedstate/SavedStateRegistry$SavedStateProvider;)V
-	public final fun runOnNextRecreation (Ljava/lang/Class;)V
```

Comment:
this method and related AutoRecreated interface are being used in android instrumented tests only